transformer_engine/pytorch/attention/dot_product_attention/utils.py

```diff
@@ -477,8 +477,10 @@ def get_attention_backend(
             if device_compute_capability < (10, 0):
                 logger.debug("Disabling FusedAttention for FP8 current scaling on arch < sm100")
                 use_fused_attention = False
-            elif cudnn_version < (9, 14, 0):
-                logger.debug("Disabling FusedAttention for FP8 current scaling with cuDNN < 9.14.0")
+            # TODO(cyanguwa): Modify the min cuDNN version supporting FP8 current scaling
+            # determinism for Blackwell
+            elif cudnn_version < (9, 18, 0):
+                logger.debug("Disabling FusedAttention for FP8 current scaling with cuDNN < 9.18.0")
```

Comment:
```suggestion
            # TODO(cyanguwa): Add min cuDNN version supporting FP8 current scaling
            # determinism for Blackwell
            else:
                logger.debug("Disabling FusedAttention for FP8 current scaling with cuDNN < 9.18.0")
```

Reply:
I think this check need not be modified as we added the CS in 9.14.0. 
The check I was looking for is to change the determinism in Blackwell.